content/project-news/reactos-in-2020.md

```diff
@@ -0,0 +1,83 @@
+---
+title:       "ReactOS in 2020"
+author:      "Extravert34"
+date:        2020-12-28
+---
+
+Despite all the turbulence, we've got quite a rich year for ReactOS.
```

Comment:
Suggestion: Despite all the turbulence, it has been quite a productive year for ReactOS.